public/main.js

```diff
@@ -1,29 +1,156 @@
 /* global $ */
 
+let teamData = []
 function fetchTeamData(path) {
   fetch(path)
     .then(res => res.json())
     .then(data => {
-      console.log(data)
+      teamData = data
+      sortExpRating(teamData)
       renderTable()
-      const tableBody = document.createElement('tbody')
-
-      data.forEach(object => {
-        const tableRow = document.createElement('tr')
-        for (let key in object) {
-          const tableData = document.createElement('td')
-          tableData.textContent = object[key]
-          tableRow.appendChild(tableData)
-        }
-        tableBody.appendChild(tableRow)
-      })
-      table.appendChild(tableBody)
+      toggleData(teamData, 'lowestTicketPrice')
+      toggleData(teamData, 'date')
+      toggleData(teamData, 'averageTicketPrice')
+      toggleData(teamData, 'experienceRating')
```

Comment:
instead of calling this function 4 times, you can put all of the keys in an array, loop over them, and call `toggleData`